File: 2328.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var n = parseInt(lines.shift()), sum = 0;
var cuts = lines.shift();
var cutsArr = cuts.split(' ');

for(let i=0; i<n; i++){
    let cut = parseInt(cutsArr[i]);
    sum += (cut - 1);
}
console.log(sum);

File: 2328.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){

    int n, cuts, sum = 0;
    cin >> n;
    while(n--){
        cin >> cuts;
        sum += (cuts - 1);
    }
    cout << sum << endl;

    return 0;
}